### ISSUE TITLE
Feature Request: Allow for marked dates style or custom style for individual dates

### ISSUE THREAD

**Issue by LeXuanKhanh** [farhoudshapouran/react-native-ui-datepicker#95]
Most calendar libs have support marked dates, I hope this lib will support this too. As for custom style for individual date, it allow for more complex marked dates style.
[CalendarPicker - Custom styling individual dates](https://github.com/stephy/CalendarPicker#custom-styling-individual-dates)

**Comment by demmyhonore** [comment]
Agree, my use case would be to be able to mark days that have appointments.

**Comment by demmyhonore** [comment]
Would be a good feature! I pivoted towards https://github.com/wix/react-native-calendars as they have marking.

**Comment by farhoudshapouran** [comment]
Thank you for using this library, I am currently working on renderDay prop and it is almost done I will publish it within some days. And you will render what you need in day cells

**Comment by farhoudshapouran** [comment]
Please upgrade to v3 and use the components prop to replace your day component with the default one. you can read more about it [here](https://github.com/farhoudshapouran/react-native-ui-datepicker?tab=readme-ov-file#custom-components) and you can found some examples [here](https://farhoudshapouran.github.io/react-native-ui-datepicker/examples)

Be careful to add new default `styles` or `classNames`. follow [basic usage](https://github.com/farhoudshapouran/react-native-ui-datepicker?tab=readme-ov-file#basic-usage)
